dis_snek/models/discord/webhooks.py

```diff
@@ -5,13 +5,14 @@
 from aiohttp import FormData
 
 from dis_snek.client.const import MISSING, Absent
-from dis_snek.client.errors import ForeignWebhookException
+from dis_snek.client.errors import ForeignWebhookException, EmptyMessageException
 from dis_snek.client.mixins.send import SendMixin
 from dis_snek.client.utils.attr_utils import define
 from dis_snek.client.utils.input_utils import _bytes_to_base64_data
 from dis_snek.models.discord.message import process_message_payload
-from dis_snek.models.discord.snowflake import to_snowflake
+from dis_snek.models.discord.snowflake import to_snowflake, to_optional_snowflake
 from .base import DiscordObject
+from ... import models
```

Comment:
So good

Reply:
That's a pycharm auto-import if I've ever seen one

Reply:
Please do.